content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,819 @@
+In this example usage of `session.NewSession` without additional argument parameters is migrated to
+`config.LoadDefaultConfig`.
+
+###### V1
+
+```go
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession()
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	// handle error
+}
+```
+
+##### Migrating from NewSession with aws.Config options.
+In this example we will show how to migrate overriding of `aws.Config` values during configuration loading. One or more
+`config.With*` helper functions can be provided to `config.LoadDefaultConfig` to override the loaded configuration
+values. In this example the AWS Region is overridden to `us-west-2` using the
+[config.WithRegion]({{< apiref "config#WithRegion" >}}) helper function.
+
+###### V1
+```go
+import "github.com/aws/aws-sdk-go/aws"
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession(aws.Config{
+	Region: aws.String("us-west-2")
+})
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(config.WithRegion("us-west-2"))
+if err != nil {
+	// handle error
+}
+```
+
+#####  Migrating from NewSessionWithOptions
+In this example we will show how to migrate overriding values during configuration loading. One or more
+`config.With*` helper functions can be provided to `config.LoadDefaultConfig` to override the loaded configuration
+values. In this example we show how to override the target profile that is used when loading the AWS SDK shared
+configuration.
+
+###### V1
+```go
+import "github.com/aws/aws-sdk-go/aws"
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSessionWithOptions(aws.Config{
+	Profile: "my-application-profile"
+})
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(config.WithSharedConfigProfile("my-application-profile"))
+if err != nil {
+	// handle error
+}
+```
+
+## Credentials & Credential Providers
+
+The [credentials]({{< apiref v1="aws/credentials" >}}) package and associated credential providers have been relocated
+to [credentials]({{< apiref "credentials" >}}) package location. The `credentials` package is Go module that is
+retrieved using `go get github.com/aws/aws-sdk-go-v2/credentials`.
+
+The AWS Credential Providers updated to provide a consistent interface for retrieving AWS Credentials. Each provider
+implements the [aws.CredentialsProvider]({{< apiref "aws#CredentialsProvider" >}}) interface, which defines a `Retrieve`
+method that returns `(aws.Credentials, error)`. [aws.Credentials]({{< apiref "aws#Credentials" >}}) is analogous to the
+AWS SDK for Go [credentials.Value]({{< apiref "aws/credentials#Value" >}}) type.
```

Comment:
Were should we callout that the `credentials.Credentials` no longer exists in v2, and that CredentialsProviders are not cached by default unless you use `CredentialCache` to wrap a provider?

Reply:
Ah yes I meant to add a section for this but did not. Will add that in a follow up push.